styles/index.css

```
@@ -0,0 +1,281 @@
+:root {
+  /* Primary color */
+  --primary-100: #3692ff;
+  --primary-200: #1967d6;
+  --primary-300: #1251aa;
+
+  /* Secondary color */
+  --secondary-50: #f9fafb;
+  --secondary-100: #f3f4f6;
+  --secondary-200: #e5e7eb;
+  --secondary-400: #9ca3af;
+  --secondary-500: #6b7280;
+  --secondary-600: #4b5563;
+  --secondary-700: #374151;
+  --secondary-800: #1f2937;
+  --secondary-900: #111827;
+
+  /* Error color */
+  --error-500: #f74747;
+
+  /* Disable color */
+  --disable-400: #9ca3af;
```

💊 제안
위에서 선언된 --secondary-400 변수가 disable 변수에서도 사용된 것이니 아래처럼 선언하시는 것이 더 의도를 나타낼 수 있습니다.
```suggestion
  --disable-400: var(--secondary-400);
```

---

미션 2때 반영하도록 하겠습니다!